index.html

```diff
@@ -353,6 +353,35 @@ <h2>
               </tr>
             </tbody>
           </table>
+          <div class="note">
+            <p>Below are few non-normative recommendations when capturing display for 
+              applications that support full screen rendering.
+            </p>
+            <p>
+              For applications that support full screen rendering, like powerpoint for example, 
+              the decision of the display surface that gets captured while in full screen mode is 
+              specific to the user agent implementation. For consistency reasons, 
+              an user agent must ensure that the display surface captured in full screen 
+              mode be unambigously mapped to the application/window where the action was triggered 
+              in the first place. 
+            </p>
+            <p>
+              The choice of <code><a>DisplayCaptureSurfaceType</a></code> and 
+              <code><a data-link-for="MediaTrackConstraintSet">logicalSurface</a></code> may 
+              further influence what gets captured. For example, when an application/window 
+              is triggered to go full screen and the DisplayCaptureSurfaceType of "monitor" 
+              is selected, the choice of the "monitor" display surface selected for
+              capture is solely left to the user agent, especially when there 
+              are mutliple options available.
```

Comment:
This example isn't particularly informative.  You might like to note that if the logical surface is being captured, then it might not make sense to follow the transition, but it might make more sense for a display surface, for which the expectation is that what is seen is captured.